.make/test.mk

```diff
@@ -328,20 +331,27 @@ $(eval PACKAGE_NAME := $(2))
 $(eval COMBINED_OUT_FILE := $(3))
 $(eval ERRORS_FILE := $(4))
 $(eval ENV_VAR := $(5))
+$(eval ALL_PKGS_COMMA_SEPARATED := $(6))
 
 @mkdir -p $(COV_DIR)/$(PACKAGE_NAME);
 $(eval COV_OUT_FILE := $(COV_DIR)/$(PACKAGE_NAME)/coverage.$(TEST_NAME).mode-$(COVERAGE_MODE))
 @$(ENV_VAR) ALMIGHTY_POSTGRES_HOST=$(ALMIGHTY_POSTGRES_HOST) \
 	go test $(PACKAGE_NAME) \
 		-v \
 		-coverprofile $(COV_OUT_FILE) \
+		-coverpkg $(ALL_PKGS_COMMA_SEPARATED) \
 		-covermode=$(COVERAGE_MODE) \
 		-timeout 10m \
 		$(EXTRA_TEST_PARAMS) \
 	|| echo $(PACKAGE_NAME) >> $(ERRORS_FILE)
 
 @if [ -e "$(COV_OUT_FILE)" ]; then \
-	tail -n +2 $(COV_OUT_FILE) >> $(COMBINED_OUT_FILE); \
+	if [ ! -e "$(COMBINED_OUT_FILE)" ]; then \
+		cp $(COV_OUT_FILE) $(COMBINED_OUT_FILE); \
+	else \
+		cp $(COMBINED_OUT_FILE) $(COMBINED_OUT_FILE).tmp; \
```

Comment:
No quite. Every time we run the unit tests (or integration tests for that matter), the `COMBINED_OUT_FILE` will be [overwritten](https://github.com/almighty/almighty-core/blob/fd204a0420e4a454832b6867f2b556d244f7f646/.make/test.mk#L370):

```Makefile
@echo "mode: $(COVERAGE_MODE)" > $(COV_PATH_UNIT)
```

`COV_PATH_UNIT` is passed to the function as `COMBINED_OUT_FILE`.